PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -283,6 +286,27 @@
                 }
             }
 
+            // if view in photo library picker controller, always mask the whole view
+            if let pickerViewController = view.nearestViewController as? UIImagePickerController {
+                maskableWidgets.append(pickerViewController.view.toAbsoluteRect(window))
+                return
+            }
+
+            if #available(iOS 14, *) {
+                if let pickerViewController = view.nearestViewController as? PHPickerViewController {
+                    maskableWidgets.append(pickerViewController.view.toAbsoluteRect(window))
+                    return
+                }
+            }
+
+            // detect any views that don't belong to the current process (likely system views)
+            if let systemRemoteView {
+                if view.isKind(of: systemRemoteView) {
+                    maskableWidgets.append(view.toAbsoluteRect(window))
+                    return
+                }
+            }
```

Comment:
do you have any examples of what a system view here is?

Reply:
most likely depends on https://github.com/PostHog/posthog-ios/pull/261#discussion_r1853700510